examples/tests/invalid/resource_captures.w

```diff
@@ -19,10 +18,6 @@ class Foo {
     log(this.mut_array.at(0));
 //        ^^^^^^^^^ Unable to reference "this.mut_array" from inflight method "test" because type MutArray<str> is not capturable
 
-    let b = this.bucket;
-//          ^^^^^^^^^^^ Unable to qualify which operations are performed on 'this.bucket' of type 'Bucket'. This is not supported yet.
-    b.put("hello", "world");
```

Comment:
I'm curious why this had to be removed

Reply:
Because the bucket was not initialized, than the fail from initialization was blocking the other errors

Reply:
Can we initialize the bucket in the `init()` section then? I think the error message here ("Unable to qualify which operations...") should still apply if the bucket is initialized

Reply:
You're right, changed